src/ray/gcs/store_client/store_client.h

```diff
@@ -69,11 +69,10 @@ class StoreClient {
   ///
   /// \param table_name The name of the table to be read.
   /// \param callback Callback that will be called after data has been received.
-  /// If the callback return `has_more == true` mean there's more data to be received.
   /// \return Status
   virtual Status AsyncGetAll(
       const std::string &table_name,
-      const SegmentedCallback<std::pair<std::string, std::string>> &callback) = 0;
+      const MultiItemCallback<std::pair<std::string, std::string>> &callback) = 0;
```

Comment:
delete `SegmentedCallback` definition if it's not used.

Reply:
fixed